Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM docker.io/python:2-alpine3.7
+
+RUN apk add --no-cache --virtual .nacl_deps \
+        build-base \
+        libressl-dev \
+        libffi-dev
+
+COPY . /sydent
+WORKDIR /sydent
+RUN cd /sydent \
+    && pip install --upgrade pip setuptools \
+    && pip install --upgrade .
```

Comment:
Email templates are not installed via pip install. You may want to copy those files to `/data` so that they can be used as relative paths in the conf file